README.md

```diff
@@ -33,7 +35,32 @@ Then install the project dependencies:
 yarn
 ```
 
-### Running the server
+### Pulling Redis' Docker
+
+Redis is used to facilitate the communication between FL4Health/Flower and
```

Comment:
maybe worth adding hyperlink to redis or redis container for context

Reply:
what do you mean by communication between FL4Health/Flower and FLorist's servers and clients? FLorist server and clients are using FL4Health/Flower under the hood so I think thats where I am getting confused. 

I was thinking redis would be a way for our FLorist Manager to fetch reported metrics from running servers and clients during runs. 

Not necessarily asking you to change the documentation but I would appreciate if you could clear up my misunderstanding so I can better understand the role of redis

Reply:
> redis would be a way for our FLorist Manager to fetch reported metrics from running servers and clients during runs.

Yes, that's what I meant, I just chose words poorly :) Looks like your understanding is correct.

Reply:
@jewelltaylor fixed